packages/nx-plugin/src/generators/configure-storybook/files/.storybook/main.ts.template

```diff
@@ -4,6 +4,9 @@ const config: StorybookConfig = {
   stories: ['../stories/**/index.stories.@(js|jsx|ts|tsx|mdx)'],
   addons: [
     '@nx/react/plugins/storybook',
+    '@storybook/addon-docs',
```

Comment:
seems this adds more than "advertised" within the PR 👀

- why do we need actions ? is anyone using that ( from my personal experiences using those might be an anti-pattern )
- what about the docs addon ?